### PR TITLE
FR-1764: Turn off the old Consent Process in contested

### PR DESCRIPTION
**Pull request by johnoneill46**
Moved following events and associated permissions:
- 'Consent Order Payment'
- 'Consent Order (No Payment)'

This means they will remain in Production BUT not appear in lower environments. A separate PR will be used to delete these files when we release "Consented in Contested"

### CODE REVIEW

**Comment by ocarleton** [comment]
@johnoneill46 Do -prod files get ignored in AAT etc the same way -nonprod aren't build on prod?

**Comment by johnoneill46** [comment]
> @johnoneill46 Do -prod files get ignored in AAT etc the same way -nonprod aren't build on prod?

Yep exactly - so we can 'toggle' things off in AAT etc but keep them live in Production

File: definitions/contested/json/CaseEvent/CaseEvent-consentedInContested-prod.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    "ShowEventNotes": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "FR_uploadConsentedOrderWithPayment",
+    "Name": "Consent Order Payment",
+    "Description": "Consent Order Payment",
+    "DisplayOrder": 23,
+    "PreConditionState(s)": "*",
```
Comment (shaunganley) [inline definitions/contested/json/CaseEvent/CaseEvent-consentedInContested-prod.json:35]: I have a feeling that removing this event will prevent the current cases in prod from being able to complete the existing consent flow in PROD assuming there will still be cases when this is released

Comment (johnoneill46) [reply definitions/contested/json/CaseEvent/CaseEvent-consentedInContested-prod.json:35]: Might be correct - will confirm with Grace what the expected behaviour of this is

Comment (johnoneill46) [reply definitions/contested/json/CaseEvent/CaseEvent-consentedInContested-prod.json:35]: So having spoken to Grace - there is only 3 cases in Production that are possibly affected by this, and Lucy believes they may be old cases anyway and they're happy to handle these manually if necessary

**Pull request merged**